tests/plugin-test/copy-plugin/helpers/getCompiler.js

```diff
@@ -8,6 +8,7 @@ module.exports = (config = {}) => {
 		context: path.resolve(__dirname, "../fixtures"),
 		entry: path.resolve(__dirname, "../helpers/enter.js"),
 		output: {
+			compareBeforeEmit: false,
```

Comment:
This is a bug of copy plugin test. It does not clean before running test cases. I will fix it in another pr